docs/overview/getting-started/intro.md

```diff
@@ -2,69 +2,53 @@
+## Building on Avalanche
 
-## Avalanche Consensus Protocol
+### Virtual Machines
 
-![Consensus Comparison](/img/Consensus-protocol-comparison.png)
+Avalanche allows developers to build Dapps through the Virtual Machine (VM) framework. VMs are 
+language agnostic and give developers unmatched levels of flexibility and customization in the
+implementation of their applications.
 
-Protocols in the Avalanche family operate through repeated sub-sampled voting. When a
-[validator](http://support.avalabs.org/en/articles/4064704-what-is-a-blockchain-validator) is
-determining whether a
-[transaction](http://support.avalabs.org/en/articles/4587384-what-is-a-transaction) should be
-accepted or rejected, it asks a small, random subset of validators whether they think the
-transaction should be accepted or rejected. If the queried validator thinks the transaction is
-invalid, has already rejected the transaction, or prefers a conflicting transaction, it replies that
-it thinks the transaction should be rejected. Otherwise, it replies that it thinks the transaction
-should be accepted.
+To learn more about VMs, click [here](/docs/overview/getting-started/virtual-machines.md).
 
-If a sufficiently large portion (_alpha_ α) of the validators sampled reply that they think the
-transaction should be accepted, the validator prefers to accept the transaction. That is, when it is
-queried about the transaction in the future, it will reply that it thinks the transaction should be
-accepted. Similarly, the validator will prefer to reject the transaction if a sufficiently large
-portion of the validators replies that they think the transaction should be rejected.
+### Subnets
 
-The validator repeats this sampling process until _alpha_ of the validators queried reply the same
-way (accept or reject) for _beta_ β consecutive rounds.
+Subnets are Avalanche's solution to scaling, and allow Avalanche to scale to handle an infinite
+amount of interoperable Dapps in its ecosystem.
 
-In the common case when a transaction has no conflicts, finalization happens very quickly. When
-conflicts exist, honest validators quickly cluster around conflicting transactions, entering a
-positive feedback loop until all correct validators prefer that transaction. This leads to the
-acceptance of non-conflicting transactions and the rejection of conflicting transactions.
+To learn more about Subnets, click [here](/docs/subnets/README.md).
 
-![How Avalanche Consensus Works](/img/howavalancheconsensusworks.png)
+## Technology
 
-It is guaranteed (with high probability based on system parameters) that if any honest validator
-accepts or rejects a transaction, all honest validators will accept or reject that transaction.
+### Consensus
 
-Learn more technical components of the Avalanche consensus protocol by reading the [white
-paper](https://arxiv.org/pdf/1906.08936.pdf).
+Avalanche uses a revolutionary proof-of-stake consensus algorithm that is scalable, robust, and
```

Comment:
Can we remove all references to a `proof of stake consensus algorithm` ?

Reply:
Removed